CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 ## [Unreleased]
 
 ### Added
+- Add the option for the external force orientation to be reperesented in the link frame with the `externalwrench` plugin (https://github.com/robotology/gazebo-yarp-plugins/pull/463).
 - Add the possibility to create different types of joints with the `linkattacher` plugin (https://github.com/robotology/gazebo-yarp-plugins/pull/461).
 
```

Comment:
Can you also add under  `### Fixed` the change related to the single/multiple mode? Thanks!

Reply:
Done in https://github.com/robotology/gazebo-yarp-plugins/pull/463/commits/c60fd0c21973623e635d0dc1c45998faa737ac6b